src/Avalonia.Base/Platform/IGeometryContextEx.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Avalonia.Platform
+{
+    public interface IGeometryContextEx : IGeometryContext
```

Comment:
In other places we added numbers. Like https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Media/IGlyphTypeface2.cs

It also should have "// TODO12 combine with IGeometryContext" comment or something.

Reply:
Done